boards/arm/atsamd21_xpro/atsamd21_xpro_defconfig

```diff
@@ -0,0 +1,21 @@
+CONFIG_ARM=y
```

Comment:
Can you add copyright/license header info

Reply:
done